src/main/java/dk/cngroup/lentils/service/StatusService.java

```diff
@@ -116,7 +123,9 @@ private void saveNewStatus(final Status status, final CypherStatus newStatus) {
     }
 
     public List<Status> getPendingCyphers(final Team team) {
-        return statusRepository.findByTeamAndCypherStatus(team, CypherStatus.PENDING);
+        List<Status> statuses = statusRepository.findByTeamAndCypherStatus(team, CypherStatus.PENDING);
+        Collections.sort(statuses, Status.getCompareByCypherStage());
```

Comment:
toto potom muzes odstranit a zavolat metodu repozitare, ktera to vrati serazene.